md/install-a-bonita-bpm-cluster.md

```diff
@@ -38,16 +38,21 @@ user at the least: `"ec2:DescribeInstances"`.
 
 In this part we will create a cluster from scratch. We will initialize the database on which the cluster will run, then we will configure nodes to run on this cluster.
 
-### <a id="create_init_bonita_db" /> Create and initialize the database for Bonita BPM Platform
+## Allow communication between servers
+Bonita cluster uses Hazelcast as the distributed cluster dispatcher layer.
+Hazelcast needs the port 5701 to be opened on all the servers that compose the cluster.
```

Comment:
Isn't this port configurable? If so, do not hard-code it.

Reply:
The port is configurable in hazelcast:
http://docs.hazelcast.org/docs/latest-development/manual/html/Setting_Up_Clusters/Other_Network_Configurations.html

I don't know how to change this port parameter in the Bonita files. If it's possible we can improve the doc. If it's not possible, provide the hardcode solution is better than ignore it in my opinion

Reply:
ok, so please just add
"Hazelcast needs the port 5701 **(by default)**..." and it's ok for me